CMake/FindArmadillo.cmake

```diff
@@ -1,41 +1,46 @@
-# - Find Armadillo
-# Find the Armadillo C++ library
-#
-# Using Armadillo:
-#  find_package(Armadillo REQUIRED)
-#  include_directories(${ARMADILLO_INCLUDE_DIRS})
-#  add_executable(foo foo.cc)
-#  target_link_libraries(foo ${ARMADILLO_LIBRARIES})
-# This module sets the following variables:
-#  ARMADILLO_FOUND - set to true if the library is found
-#  ARMADILLO_INCLUDE_DIRS - list of required include directories
-#  ARMADILLO_LIBRARIES - list of libraries to be linked
-#  ARMADILLO_VERSION_MAJOR - major version number
-#  ARMADILLO_VERSION_MINOR - minor version number
-#  ARMADILLO_VERSION_PATCH - patch version number
-#  ARMADILLO_VERSION_STRING - version number as a string (ex: "1.0.4")
-#  ARMADILLO_VERSION_NAME - name of the version (ex: "Antipodean Antileech")
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file Copyright.txt or https://cmake.org/licensing for details.
 
-#=============================================================================
-# Copyright 2011 Clement Creusot <creusot@cs.york.ac.uk>
-#
-# Distributed under the OSI-approved BSD License (the "License");
-# see accompanying file Copyright.txt for details.
-#
-# This software is distributed WITHOUT ANY WARRANTY; without even the
-# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-# See the License for more information.
-#=============================================================================
-# (To distribute this file outside of CMake, substitute the full
-#  License text for the above reference.)
+#[=======================================================================[.rst:
+FindArmadillo
+-------------
 
+Find the Armadillo C++ library.
+Armadillo is library for linear algebra & scientific computing.
```

Comment:
```suggestion
Armadillo is a library for linear algebra & scientific computing.
```
:)

Reply:
Ah, thanks, though for once I can't take credit for [this typo](https://github.com/Kitware/CMake/blob/master/Modules/FindArmadillo.cmake#L9) myself ;)